packages/story-editor/src/components/panels/design/warning/warning.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { __ } from '@web-stories-wp/i18n';
+import { useEffect } from '@web-stories-wp/react';
+import { Text, Icons, useLiveRegion } from '@web-stories-wp/design-system';
+
+const WarningContainer = styled.div`
+  display: flex;
+  gap: 8px;
+  width: 100%;
+  margin: 14px auto;
+  border: 1px solid ${({ theme }) => theme.colors.border.defaultHover};
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  padding: 8px;
+  color: ${({ theme }) => theme.colors.fg.primary};
+  font-weight: ${({ theme }) => theme.typography.weight.regular};
+`;
+const WarningIcon = styled(Icons.WarningLow)`
+  width: 32px;
+  height: 100%;
+  color: #f4b844;
```

Comment:
If this is a new color, can you add it to the design system? Or if it's similar to another color we've got in there, let's use that instead to keep the color palette tight(ish).

Reply:
I couldn't find any thing of the sort, happy to add.

Reply:
I added `warning: {yellow: #f4b844}` Don't know if that is how we want to add this. We didn't have anything similar so I am unsure. Do I need to add something to Figma as well?

Reply:
you don't need to add to figma - the theme colors populate in storybook: https://google.github.io/web-stories-wp/storybook/?path=/story/designsystem-colors--default 

I think that this yellow goes with the `status` section of colors, keeping colors organized by purpose can allow for easier updates. If we call it yellow and then maybe someday (just an example) the designs change the warning color to be an orange, now that reads wrong because the yellow is actually orange and we'd have to update the instances of that instead of just the theme color.